### PR TITLE
minor cleanup: fixing or silencing flake8 errors

### PR DESCRIPTION
**Pull request by rwgk**
Fixing errors that appeared after a flake8 version bump.

### CODE REVIEW

**Review by bstaletic — APPROVED** [review]
Maybe we should just disable B015? Either way, approved.

**Comment by rwgk** [comment]
Thanks @bstaletic ! I'll wait a bit to see if we can get the CI green (waiting for @henryiii's cmake pinning).

I think in general B015 is useful. I feel disabling it completely would be a loss, I'd go there only if we had to put in a whole bunch more of the noqa comments.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -596,6 +600,9 @@ jobs:
 
     - name: Update CMake
       uses: jwlawson/actions-setup-cmake@v1.4
+      with:
+        cmake-version: 3.19.1
+
```
Comment (bstaletic) [inline .github/workflows/ci.yml:605]: CI is complaining about an extra empty line here.

Comment (henryiii) [reply .github/workflows/ci.yml:605]: Are we interested in enabling pre-commit.ci? It would automatically fix things like this for us.

Comment (henryiii) [reply .github/workflows/ci.yml:605]:
```suggestion
```

File: tests/test_local_bindings.py [hunk 2]
```diff
@@ -193,7 +193,7 @@ def test_stl_caster_vs_stl_bind(msg):
     v2 = [1, 2, 3]
     assert m.load_vector_via_caster(v2) == 6
     with pytest.raises(TypeError) as excinfo:
-        cm.load_vector_via_binding(v2) == 6
+        cm.load_vector_via_binding(v2)
```
Comment (henryiii) [inline tests/test_local_bindings.py:196]: I think `B015` helps in cases like this. It clarifies the test is really about the call, and not the comparison, which never happens.

**Pull request merged**